src/libraries/Microsoft.PowerFx.Core/Public/Types/ControlType.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.PowerFx.Core.Types;
+
+namespace Microsoft.PowerFx.Types
+{
+    /// <summary>
+    /// Representing UI control type.
+    /// </summary>
+    public class ControlType : AggregateType
```

Comment:
> `: AggregateType`

This will probably not work, as it requires `GetAllNames()` which (I believe) will not work for a control DType.

Reply:
The implementation was dropped.